x-pack/filebeat/module/cisco/asa/test/additional_messages.log-expected.json

```diff
@@ -838,7 +850,7 @@
         "input.type": "log",
         "log.level": "informational",
         "log.offset": 2298,
-        "network.bytes": 0,
+        "network.bytes": "0",
```

Comment:
It seems that networ.bytes was moved from long to string, might need to convert these

Reply:
Just committed a fix for this (added conversion for network.bytes in the conversion section near end of pipeline).